bob-access/src/lib.rs

```diff
@@ -114,10 +105,11 @@ where
         debug!("credentials: {:#?}", credentials);
         if let Err(e) = self.authenticator.check_credentials(credentials) {
             warn!("Unauthorized request: {:?}", e);
-            let error = Box::new(Error::unauthorized_request()) as Self::Error;
-            Box::pin(futures::future::ready(Err(error))) as Self::Future
+            // let error = Box::new(Error::unauthorized_request()) as Self::Error;
+            // Box::pin(futures::future::ready(Err(error))) as Self::Future
+            todo!()
```

Comment:
Unauthorized error code must somehow be returned here

Reply:
Related to #217 request